spec/unit/rust-crypto/backup.spec.ts

```diff
@@ -8,10 +8,9 @@ import * as testData from "../../test-utils/test-data";
 import * as TestData from "../../test-utils/test-data";
 import { IKeyBackup } from "../../../src/crypto/backup";
 import { IKeyBackupSession } from "../../../src/crypto/keybackup";
-import { defer } from "../../../src/utils";
 import { RustBackupManager } from "../../../src/rust-crypto/backup";
 
-describe("PerSessionKeyBackupDownloader", () => {
+describe("Import keys from backup", () => {
```

Comment:
isn't it exporting keys to backup?

Reply:
Good catch, look like I was confused by my recent work on backup import